.github/ISSUE_TEMPLATE/Standard-Feature-Issue-Template.md

```diff
@@ -28,24 +28,21 @@ devchat: < If HL Engineering requires notification instead of the end user or if
 < list of subject domain experts for this domain. If the subject expert from the end-user perspective is not the author, an end-user expert should be listed here. The Design, Dev, and QA teams should add experts here as appropriate. >
 
 ## Progress
-### Ready for Planning
+### Reviewed Issues - completed by SM & Team Leads
 - [ ] Story and Business Value complete
 - [ ] Initial Acceptance Criteria Defined
 - [ ] The person who will Approve story identified
 - [ ] Proposed Solution provided
 
-### Ready for Review
-- [ ] Testing implemented and passed
-
-### Ready for Pre-Production QA
+### Implemented Issue - completed by Developer assigned
 - [ ] Issue scored
 - [ ] Acceptance criteria met - boxes checked
 - [ ] Announcement message created
 
-### Ready for SM Approval
+### Tested Issue - Completed by QA assigned
 - [ ] Manual Acceptance Test Results included in Comment
 
-### Ready for PO Approval
+### Deploy-able Issue - completed by SM
 - [ ] Announcement message approved
 - [ ] Issue complete and correct in all required sections
-- [ ] All checkboxes checked
+- [ ] Issue requirements met
```

Comment:
What does this mean exactly? Is this different than the acceptance criteria?

Reply:
I mean that not only are the boxes checked but we actually met the acceptance criteria, have everything in the Issue that's required to be there, etc. Maybe that's redundant with "Issue complete and correct in all required sections" and I can change "Issue requirements met" with "Acceptance Criteria met"